Gem/Code/Source/Components/UI/UiMatchPlayerCoinCountsComponent.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include <MatchPlayerCoinsBus.h>
 #include <AzCore/Serialization/EditContext.h>
+#include <LyShine/Bus/UiElementBus.h>
```

Comment:
was this not working before? (looks like it's already being called on L39/40)

Reply:
It didn't compile for me. Probably a difference between using unity builds or non-unity builds.